turbo/app/snapshots_cmd.go

```diff
@@ -780,38 +781,26 @@ func doRetireCommand(cliCtx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	logger.Info("Compute commitment")
-	if err = db.Update(ctx, func(tx kv.RwTx) error {
-		if casted, ok := tx.(kv.CanWarmupDB); ok {
-			if err := casted.WarmupDB(false); err != nil {
-				return err
-			}
-		}
-		ac := agg.MakeContext()
-		defer ac.Close()
-		sd := libstate.NewSharedDomains(tx, logger)
-		defer sd.Close()
-		if _, err = sd.ComputeCommitment(ctx, true, sd.BlockNum(), ""); err != nil {
-			return err
-		}
-		if err := sd.Flush(ctx, tx); err != nil {
-			return err
-		}
-		return err
-	}); err != nil {
-		return err
-	}
 
 	logger.Info("Prune state history")
 	for i := 0; i < 1; i++ {
 		if err := db.UpdateNosync(ctx, func(tx kv.RwTx) error {
 			ac := agg.MakeContext()
 			defer ac.Close()
-			if ac.CanPrune(tx) {
-				if err = ac.PruneSmallBatches(ctx, time.Hour, tx); err != nil {
-					return err
-				}
+
+			logEvery := time.NewTicker(30 * time.Second)
+			defer logEvery.Stop()
+
+			cc := context.Background()
+			err = ac.ResetPruneProgress(tx) // reset previous prune progress to make it a "clean run"
```

Comment:
Don’t understand why need reset here. Why can’t continue pruning? Does it mean Erigon also can’t continue pruning after restart?